deployment/terraform/variables.tf

```diff
@@ -0,0 +1,191 @@
+variable "project" {
+  default = "Open Apparel Registry"
+}
+
+variable "environment" {
+  default = "Staging"
+}
+
+variable "aws_region" {
+  default = "us-east-1"
+}
+
+variable "aws_availability_zones" {
+  default = ["us-east-1a", "us-east-1b"]
+}
+
+variable "aws_key_name" {}
+
+variable "r53_private_hosted_zone" {}
+
+variable "r53_public_hosted_zone" {}
+
+variable "cloudfront_price_class" {}
+
+variable "vpc_cidr_block" {
+  default = "10.0.0.0/16"
+}
+
+variable "external_access_cidr_block" {}
+
+variable "vpc_private_subnet_cidr_blocks" {
+  default = ["10.0.1.0/24", "10.0.3.0/24"]
+}
+
+variable "vpc_public_subnet_cidr_blocks" {
+  default = ["10.0.0.0/24", "10.0.2.0/24"]
+}
+
+variable "bastion_ami" {}
+
+variable "bastion_instance_type" {}
+
+variable "fargate_cpu" {
+  default = "256"
+}
+
+variable "fargate_memory" {
+  default = "512"
+}
+
+variable "image_tag" {}
+
+variable "app_port" {
+  default = "8080"
+}
+
+variable "google_geocoding_api_key" {}
+
+variable "django_secret_key" {}
+
+variable "app_count" {
+  default = "2"
```

Comment:
I think that we can safely make this 1, or override it for staging so that `app_count=1`, to save on costs.